proxies/shared/policies/Quota.Global.xml

```diff
@@ -12,7 +12,7 @@ For more information on rate limiting, see the following resources:
     <DisplayName>Quota.Global</DisplayName>
     <Interval>1</Interval>
     <TimeUnit>minute</TimeUnit>
-    <Allow count="6000"/>

```

Comment:
Was this meant to be a permanent change?

Reply:
Nope, will be removed, just holding this here because I don't know if Phil approves of the approach we took yet